tools/db_bench_tool.cc

```diff
@@ -2502,6 +2506,7 @@ class Benchmark {
   int64_t merge_keys_;
   bool report_file_operations_;
   bool use_blob_db_;  // Stacked BlobDB
+  bool read_operands_;  // read via GetMergeOperands()
```

Comment:
Is this need initialized to false somewhere?

Reply:
Thanks, I forgot. Done.